src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
+
+type FindMovieProps = {
+  setAddedMovies: (addedMovies: Movie[]) => void;
+  addedMovies: Movie[];
+};
+
+export const FindMovie: React.FC<FindMovieProps> = ({
+  setAddedMovies,
+  addedMovies,
+}) => {
+  const [query, setQuery] = useState('');
+  const [foundMovie, setFoundMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const handleChangeQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    setIsError(false);
+  };
+
+  const handleOnClickSearchMovie = (
+    event: React.MouseEvent<HTMLButtonElement>,
+  ) => {
+    event.preventDefault();
+    if (!query) {
+      return;
+    }
+
+    setIsLoading(true);
+    getMovie(query)
+      .then(movie => {
+        if ('Error' in movie) {
+          setIsError(true);
+        } else {
```

Comment:
```suggestion
        if ('Error' in movie) {
          setIsError(true);
        }
```
else it's not clean code